### PR TITLE
PMM-7279 param units

### PR DESCRIPTION
**Pull request by artemgavrilov**
[PMM-7279](https://jira.percona.com/browse/PMM-7279)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1376

- [x] https://github.com/percona-platform/saas/pull/29
- [x] https://github.com/percona/pmm-managed/pull/613

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -11,3 +11,7 @@ required = [
 [[constraint]]
   name = "github.com/percona/pmm"
   branch = "PMM-2.0"
+
+[[constraint]]
+name = "github.com/percona-platform/saas"
+branch = "PMM-7279-param-units"
```
Comment (palash25) [inline Gopkg.toml:17]: missing newline

Comment (artemgavrilov) [reply Gopkg.toml]: Fixed

Comment (artemgavrilov) [inline Gopkg.toml:17]: TODO

File: testdata/ia/template.yaml [hunk 2]
```diff
@@ -12,6 +12,10 @@ templates:
           type: float
           range: [0, 100]
           value: 80
+        - name: duration
+          summary: another test summary
+          unit: s
+          type: float
```
Comment (artemgavrilov) [inline testdata/ia/template.yaml:18]: Parameter without range and default value

Comment (BupycHuk) [reply testdata/ia/template.yaml:18]: isn't it better to write this as summary?

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Some API tests are failing, please check https://pmm.cd.percona.com/job/pmm2-api-tests/5424/console

File: testdata/ia/template.yaml (hunk 2 above)
Comment (artemgavrilov) [reply testdata/ia/template.yaml]: Better, fixed 👍

**Review by BupycHuk — APPROVED** [review]

File: Gopkg.toml (hunk 1 above)
Comment (palash25) [reply Gopkg.toml:17]: the saas PR was merged please update this

**Review by palash25 — APPROVED** [review]

File: Gopkg.toml (hunk 1 above)
Comment (artemgavrilov) [reply Gopkg.toml]: Done

**Pull request merged**